crates/astria-sequencer-relayer/src/relayer.rs

```diff
@@ -127,6 +127,18 @@ impl Relayer {
             }
         };
 
+        let mut namespaces_to_tx_count = std::collections::HashMap::new();
+        sequencer_block.rollup_txs.iter().for_each(|(ns, txs)| {
+            namespaces_to_tx_count.insert(*ns, txs.len());
+        });
+
+        info!(
+            sequencer_block = height,
+            proposer = ?sequencer_block.header.proposer_address,
+            namespaces_to_tx_count = ?namespaces_to_tx_count,
```

Comment:
Using rust debug for tracing fields always ends up being messy.

Alternative suggestion:

1. create the map as above, but write the result of `serde_json::to_string` into the field;
2. write a serialization wrapper and serialize the blocks nicely directly using a `Serialize` impl. You can Almost directly use the example for `MyMap` here: https://serde.rs/impl-serialize.html#serializing-a-sequence-or-map

IMO the second option would be cleanest and avoid the extra allocation on every submission (which however would probably not be so bad).